### PR TITLE
Update Tomcat distro url

### PR DESCRIPTION
**Pull request by henrikhgbg**
The old url is not valid anymore - updated to the latest patch version.

### CODE REVIEW

File: uaa-server/build.gradle [hunk 1]
```diff
@@ -50,7 +50,7 @@ cargo {
     local {
         outputFile = file("$buildDir/uaa-server.log")
         installer {
-            installUrl = 'https://www-us.apache.org/dist/tomcat/tomcat-8/v8.5.35/bin/apache-tomcat-8.5.35.zip'
+            installUrl = 'https://www-us.apache.org/dist/tomcat/tomcat-8/v8.5.38/bin/apache-tomcat-8.5.38.zip'
```
Comment (mateus-batista) [inline uaa-server/build.gradle]: URL not working.
Here's a working one: ```https://www-us.apache.org/dist/tomcat/tomcat-8/v8.5.40/bin/apache-tomcat-8.5.40.zip```

**Review by mateus-batista — CHANGES_REQUESTED** [review]

File: uaa-server/build.gradle (hunk 1 above)
Comment (henrikhgbg) [reply uaa-server/build.gradle]: I changed to the archive site instead - should be more stable.

Comment (midare16) [reply uaa-server/build.gradle]: [https://archive.apache.org/dist/tomcat/](https://archive.apache.org/dist/tomcat/) will be better

**Comment by spencergibb** [comment]
superceded by #6

**Pull request closed without merging**